subsys/net/lib/lwm2m_client_utils/fota/settings.c

```diff
@@ -66,6 +66,11 @@ static struct settings_handler fota_settings = {
 int fota_settings_init(void)
 {
 	int err;
+	static bool init;
```

Comment:
This is just to prevent calling `settings_subsys_init` multiple times, right?

Reply:
Yes.